src/main/java/org/springframework/shell/core/JLineShellComponent.java

```diff
@@ -237,4 +238,9 @@ protected String getProductName() {
 	protected String getVersion() {
 		return version;
 	}
-}
\ No newline at end of file
+
+	public CommandLine getCommandLine() {
```

Comment:
If this is just for testing then you likely want to make it package protected (ie: no modifier) and use the `@VisibleForTesting` annotation. Usually don't want to expose the internals of a class.